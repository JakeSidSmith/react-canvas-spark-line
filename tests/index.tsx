import React from 'react';
import renderer from 'react-test-renderer';
import SparkLine from '../src';

describe('SparkLine', () => {

  it('should render a canvas with the provided width and height', () => {
    const tree = renderer.create(
      <SparkLine
        width={80}
        height={20}
        data={[1, 2, 3]}
        color="red"
      />
    );
    expect(tree).toMatchSnapshot();
  });

});
