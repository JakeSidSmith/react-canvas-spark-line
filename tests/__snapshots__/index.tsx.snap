// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SparkLine should render a canvas with the provided width and height 1`] = `
<canvas
  height={40}
  style={
    Object {
      "height": 20,
      "width": 80,
    }
  }
  width={160}
/>
`;
